services/product-service/tests/test_dependencies.py

```
@@ -0,0 +1,44 @@
+    with pytest.raises(HTTPException) as exc:
+        require_admin(x_user_role=role)
+
+    assert exc.value.status_code == 403
+
+
+def test_verify_internal_service_accepts_matching_token(monkeypatch: pytest.MonkeyPatch) -> None:
+    """내부 서비스 토큰이 설정값과 일치하면 인증이 통과되는지 확인한다."""
+    monkeypatch.setenv("INTERNAL_SERVICE_TOKEN", "expected")
+    get_settings.cache_clear()
+
+    assert verify_internal_service(x_internal_token="expected") is None
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to argument: "x_internal_token"

(S106)